packages/tendermint-rpc/src/client.spec.ts

```diff
@@ -130,14 +130,31 @@ function defaultTestSuite(rpcFactory: () => RpcClient, adaptor: Adaptor, expecte
     client.disconnect();
   });
 
+  it("can get validators", async () => {
+    pendingWithoutTendermint();
+    const client = await Client.create(rpcFactory(), adaptor);
+    const response = await client.validators();
+
+    expect(response).toBeTruthy();
+    expect(response.blockHeight).toBeGreaterThanOrEqual(1);
+    expect(response.count).toBeGreaterThanOrEqual(1);
+    expect(response.total).toBeGreaterThanOrEqual(1);
+    expect(response.validators.length).toBeGreaterThanOrEqual(1);
+    expect(response.validators[0].address!.length).toEqual(20);
```

Comment:
Any clue if/why this address can be unset? Would be good to have a note in `export interface Validator {` if you do

Reply:
Checking the type from the codec it doesn't seem to be optional.

Reply:
But pubkey does!